spec/takeaway_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'takeaway'
+
+RSpec.describe Takeaway do
+
+  context "#show_menu" do
+    it "should print the menu" do
+      expect(subject.show_menu).to be_kind_of Menu
+    end
+  end
+
```

Comment:
I'm not sure what a better practice would be, but I'm not sure this is necessarily the best test for your 'show_menu' method. I understand that, given the current objectives of your program, and given the fact that @menu initializes with 'Menu.new' (i.e. an instance of the method class), if your program works successfully, this test will pass. But I feel like this isn't testing the show_menu in as much isolation as it could be (i.e. you're testing not only that show_menu returns @menu, but also that @menu is an instance of the Menu class). Additionally, because it is possible to initialize the Takeaway class with menu equal to something other than the Menu.class, if you end up changing the name of the menu class or making some other change there, it will also mistakenly throw off your show_menu class